web/text_layer_builder.js

```diff
@@ -314,30 +319,37 @@ var TextLayerBuilder = (function TextLayerBuilderClosure() {
      */
     _bindMouse: function TextLayerBuilder_bindMouse() {
       var div = this.textLayerDiv;
+      var self = this;
       div.addEventListener('mousedown', function (e) {
+        if (self.enhanceTextSelection && self.textLayerRenderTask) {
+          self.textLayerRenderTask.expandTextDivs(true);
+        }
```

Comment:
Then I'd assume that we also want to change https://github.com/mozilla/pdf.js/blob/master/web/text_layer_builder.js#L67-L69 to something like:
```javascript
if (!this.enhanceTextSelection) {
  var endOfContent = document.createElement('div');
  endOfContent.className = 'endOfContent';
  this.textLayerDiv.appendChild(endOfContent);
}
```